common/src/main/scala/org/neo4j/spark/util/Validations.scala

```diff
@@ -53,7 +53,7 @@ case class ValidateSparkVersion(supportedVersions: String*) extends Validation {
     val splittedVersion = sparkVersion.split("\\.")
 
     ValidationUtil.isTrue(
-      sparkVersion(0) == "UNKNOWN" || supportedVersions
+      supportedVersions
```

Comment:
I think that maybe we can improve it in this way.
```suggestion
      sparkVersion == "UNKNOWN" || supportedVersions
```

If for some reason we cannot retrieve the version we don't want to block the user.